aea/test_tools/test_cases.py

```diff
@@ -479,7 +479,7 @@ def send_signal_and_wait(s: signal.Signals) -> None:
         if system_name == "Windows":  # pragma: no cover
             # ctrl-c event will be handled with stdin in Windows
             consume(map(lambda p: p.stdin.close(), active_agents()))
-            signals = [signal.CTRL_C_EVENT, signal.CTRL_BREAK_EVENT, signal.SIGTERM]  # type: ignore
+            signals = [signal.CTRL_BREAK_EVENT, signal.CTRL_C_EVENT, signal.SIGTERM]  # type: ignore
```

Comment:
Can you expand why the order matters here?

Reply:
@DavidMinarsch 
just read documentation and checked out on tests cases on autonomy. and it helped